tests/unit/loadDojoConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, it } from 'intern!bdd';
+import * as assert from 'intern/chai!assert';
+import loadDojoConfig from '../../src/loadDojoConfig';
+
+describe('loadDojoConfig', () => {
+	it('should load the specified module if it exists', () => {
+		const config = loadDojoConfig('tests/support/mocks/dojo.config.js');
```

Comment:
Yes. The idea is that there should only be one config, and `webpack.config.js` would pull in just this one file. Honestly, passing a filename to the function instead of hard-coding a path serves in part to facilitate testing (so that a config does not need to be generated at the root of the cwd and then removed after testing).